woocommerce/woocommerce-ios
# Dynamic Dashboard: New update for Blaze card

Closes: #12488


## Description


This implements the following design:

<img width="583" alt="Screenshot 2024-04-29 at 09 20 13" src="https://github.com/woocommerce/woocommerce-ios/assets/266376/58684c7c-8ef3-427c-a07d-77198b17832d">

Some decisions made in this PR since there's no direct design equivalent:
1. In the case of only 1 existing campaign, the view all campaign buttons will simply say "View all campaigns", instead of "View all 1 campaign". The latter sounds awkward as a sentence.


## Testing instructions


1. Use a site eligible for Blaze but with no Blaze campaign,
2. Open dashboard, check Blaze card,
3. Ensure the layout matches the design above or screenshot below for "No campaigns state"
4. Switch to a site that has multiple Blaze campaigns,
5. Open dashboard, check Blaze card,
6. Ensure the layout matches the design above or screenshot below for "View campaigns state"

## Screenshots


| No campaigns state | View campaigns state |
|-|-|
| ![Simulator Screenshot - iPhone 15 Pro - 2024-04-29 at 14 04 13](https://github.com/woocommerce/woocommerce-ios/assets/266376/85ff292d-b0a9-44df-a3d7-6c1b485259ab) | ![Simulator Screenshot - iPhone 15 Pro - 2024-04-29 at 14 03 52](https://github.com/woocommerce/woocommerce-ios/assets/266376/bb13c4f5-fe64-4c8c-a6f0-b7eedde95122)|



---
